src/ensembl/tsconfig.json

```diff
@@ -9,8 +9,8 @@
     "module": "esnext",
     "moduleResolution": "node",
     "noEmit": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
```

Comment:
Changed this so that we do not **fail** the build if there are unused variables in code. Unused variables should be pointed out by the linter and dealt with at that stage.